addon/components/new-osf-navbar/template.hbs

```diff
@@ -0,0 +1,75 @@
+<div class="new-osf-navbar-styling">
+    <nav class="navbar navbar-inverse" id="navbarScope">
+        <div class="container">
+            <div class="navbar-header">
+                {{!Toggle navigation button for xs screens}}
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".secondary-navigation" {{action 'closeSearch'}}>
+                    <span class="sr-only">Toggle navigation</span>
```

Comment:
In general, we've been encouraging all ember apps to use `ember-i18n` out of the box: it's easier to build that in up front than to add it later.

Would it be considered within scope to request translation support before this PR goes forward?